src/components/pages/PendingApplications/MenteeModal.js

```diff
@@ -0,0 +1,123 @@
+    <>
+      <div className="profile-intro">
+        <div className="image-container">
+          <img
+            src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
+            alt="I can do this if I work hard enough and practice my coding skills"
+          />
+        </div>
+        <div className="profile-intro-description">
+          <h2>
+            {applicant.first_name} {applicant.last_name}
+          </h2>
+          <div className="preferences">
+            <div>
+              <p>Interested in: </p>
+            </div>
+            <div className="mentee-tag">
+              <Tag color={'purple'}>{applicant.tech_stack}</Tag>
+            </div>
+          </div>
+        </div>
+      </div>
+      <List size="small">
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Role</p>
+          </div>
+          <div className="list-item-column">
+            <p>Mentee</p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Email:</p>
+          </div>
+          <div className="list-item-column">
+            <p>{applicant.email}</p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Location:</p>
+          </div>
+          <div className="list-item-column">
+            <p>
+              {applicant.state} {applicant.country}
+            </p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <p className="list-item-column">Membership Criteria:</p>
+          <div className="list-item-column">
+            {applicant.formerly_incarcerated === true ||
+            applicant.low_income === true ||
+            applicant.underrepresented_group === true ? (
+              <ul>
+                {applicant.formerly_incarcerated === true ? (
+                  <li>Formerly Incarcerated</li>
+                ) : null}
+                {applicant.low_income === true ? <li>Low Income</li> : null}
+                {applicant.underrepresented_group === true ? (
+                  <li>Belongs to underrepresented group</li>
+                ) : null}
+              </ul>
+            ) : (
+              <p>Not available</p>
+            )}
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <p className="list-item-column">Convictions:</p>
+          <div className="list-item-column">
+            <ul>
+              {applicant.formerly_incarcerated === true ? (
+                applicant.convictions.split(', ').map(conviction => {
+                  return <li key={conviction}>{conviction}</li>;
+                })
+              ) : (
+                <p>None</p>
+              )}
+            </ul>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
```

Comment:
If this class name is being applied to every list item, I would suggest, instead, to use CSS selectors to accomplish the same thing. If someone later decides to change or remove these classes / styles, they will have to change remove this from each instance of the list item. Leverage the ability of CSS to apply classes when necessary